src/cockpit/channels/filesystem.py

```diff
@@ -289,13 +290,19 @@ def do_close(self):
         self.close()
 
 
+class Follow(enum.Enum):
+    NO = False
+    YES = True
```

Comment:
What's wrong with a simple boolean kwarg (enforced with `*`)? `follow=False` should convey exactly the same information as `Follow.NO`

Reply:
Good question!  This was my first approach.  Unfortunately, the `getattrs()` function has its type defined here:

```python
    getattrs: 'Callable[[int, str, Follow], JsonDocument]'
```

and `Callable` has no support for kwargs.  That presented three possibilities:
 - `Enum` (as I did)
 - [a callable protocol](https://mypy.readthedocs.io/en/stable/protocols.html#callback-protocols)
 - `#noqa`

I chose the `Enum` way.  I also tried passing the `stat` function itself as an argument (default value `lstat`), but this also turned into typing pain.

There might be a better way...

Reply:
Ah, so this isn't just a whim. No worries then.